java/RSPEC-131.java

```diff
@@ -0,0 +1,11 @@
+class SwitchNoDefault {
+    void test() {
+        switch (myVariable) {
+            case 1:
+                foo();
+                break;
+            case 2:
```

Comment:
RSPEC-128: Switch without break
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._